### PR TITLE
Cheyenne to Derecho Script and Documentation Update

### PR DESCRIPTION
**Pull request by scrasmussen**
Updating scripts and documentation to reflect NCAR's move from Cheyenne to Derecho clusters. Need to find the paths to UFS_RTs on Derecho and then the PR will be moved from `draft` to `ready for review`.

- [x] update Cheyenne scratch paths

#### Paths that were updated
``` Bash
scm/etc/scripts/GABLS3_LSM.ncl:;  gl_dir = "/glade/scratch/damico/GLDAS_GABLS3/"
scm/etc/scripts/GABLS3_LSM_RUC.ncl:;  gl_dir = "/glade/scratch/damico/GLDAS_GABLS3/"
scm/etc/scripts/GABLS3_LSM_NoahMP.ncl:;  gl_dir = "/glade/scratch/damico/GLDAS_GABLS3/"
scm/doc/TechGuide/chap_cases.rst:   ./UFS_forcing_ensemble_generator.py -d /glade/scratch/epicufsrt/GMTB/CCPP-SCM/UFS_RTs/control_c192/ -sc --C_RES 192 -dt 360  -n control_c
192 -lons 300 -lats 34
scm/doc/TechGuide/chap_cases.rst:   ./UFS_forcing_ensemble_generator.py -d /glade/scratch/epicufsrt/GMTB/CCPP-SCM/UFS_RTs/control_c384/ -sc --C_RES 384 -dt 225 -n control_c3
84 -lons 300 300 300 300 -lats 34 35 35 37
scm/doc/TechGuide/chap_cases.rst:   ./UFS_forcing_ensemble_generator.py -d /glade/scratch/epicufsrt/GMTB/CCPP-SCM/UFS_RTs/control_p8/ -sc --C_RES 96 -dt 720 -n control_p8 -l
onl 300 320 -latl 40 50 -nens 10 -sdf SCM_GFS_v17_p8
```

### CODE REVIEW

File: scm/doc/TechGuide/chap_cases.rst
Comment (mkavulich) [inline scm/doc/TechGuide/chap_cases.rst:511]: It looks like this directory was never ported over from Cheyenne before it was scrubbed. @dustinswales Does it make sense to grab some examples from the WM baselines Derecho and ask EPIC to stage them in a similar Derecho directory? Does it matter which tests we use?

File: scm/doc/TechGuide/chap_cases.rst
Comment (mkavulich) [inline scm/doc/TechGuide/chap_cases.rst:523]: We probably don't want to use these directories since they are scrubbed on a semi-regular basis

File: scm/etc/scm_qsub_example.py
Comment (mkavulich) [inline scm/etc/scm_qsub_example.py:6]: Has this been tested on Derecho? I don't think I ever tried out this example script in my Derecho testing.

File: scm/etc/scripts/GABLS3_LSM.ncl
Comment (mkavulich) [inline scm/etc/scripts/GABLS3_LSM.ncl:8]: We're still supporting NCL scripts? 😱

File: scm/doc/TechGuide/chap_cases.rst
Comment (scrasmussen) [reply scm/doc/TechGuide/chap_cases.rst]: Changed to more generic `/glade/derecho/scratch/epicufsrt/FV3_RT/` which does exist

File: scm/doc/TechGuide/chap_cases.rst
Comment (scrasmussen) [reply scm/doc/TechGuide/chap_cases.rst]: Changed to more generic `./UFS_forcing_ensemble_generator.py -d [path_to_regression_tests_output]/control_c192_intel/ -sc --C_RES 192 -dt 360  -n control_c192 -lons 300 -lats 34`

File: scm/etc/scm_qsub_example.py
Comment (scrasmussen) [reply scm/etc/scm_qsub_example.py:6]: Generalized this script more, testing right now, thanks for your input!

File: scm/etc/scripts/GABLS3_LSM.ncl
Comment (scrasmussen) [reply scm/etc/scripts/GABLS3_LSM.ncl:8]: Lol, I'm not sure if we officially support them. I didn't try them, only changed `Cheyenne -> Derecho` and the editor automatically cleaned up the trailing whitespace

Comment (grantfirl) [reply scm/etc/scripts/GABLS3_LSM.ncl:8]: I've only ever run these scripts once on my local machine. They're not really meant to be run again and are saved in the repo in case there is a mistake in the forcing file and need to be regenerated. I don't think we need to "support" this to anyone but ourselves. This goes for most of the the "leftover" scripts in etc/scripts. This should probably be cleaned up at some point.

**Review by grantfirl — APPROVED** [review]
Looks good to me. Please pull in the latest main before we push the merge button.

**Comment by scrasmussen** [comment]
> Looks good to me. Please pull in the latest main before we push the merge button.

Thanks for the review, just pulled in the latest main 👍

**Pull request merged**
